### DISCUSSION TITLE
Negative numbers with thousand separator and data-maska-reversed

### DISCUSSION THREAD

**Discussion by maringomori** [beholdr/maska#116] [Q&A]
Hey,
I tried to make a mask which handles negative numbers with thousand separator, but if I add the data-maska-reversed its not repeating anymore.
Can you help me out what I'm doing wrong? 

<input
  v-maska
  data-maska="10 00"
  data-maska-tokens="1:-:optional|0:\d:repeated"
  data-maska-reversed 
>

If the reversed is not there it's working expected, but I'm planning to use it for money so I need it to be reversed

<input
  v-maska
  data-maska="10 00"
  data-maska-tokens="1:-:optional|0:\d:repeated"
>

Thanks for help in advance

**Comment by beholdr** [comment]
I'm not sure it possible with current `reversed` implementation. But you can try to use `postProcess` hook to format number. Something like this:

``` html
const options = {
  postProcess: (val) => (val && val !== '-') ? new Intl.NumberFormat().format(val) : ''
}

<input v-maska:[options] data-maska="-0" data-maska-tokens="-:-:optional|0:[0-9]:multiple">
```
